### ISSUE TITLE
Carrying multiple animals with piggyback crashes.

### ISSUE THREAD

**Issue by bluelightning32** [capnkirok/animaniamod#326]
### Expected Behavior
Carrying multiple animals with piggyback does not crash

### Actual Behavior
The server hangs, then the watchdog kills it.

### Steps to Reproduce
Despite multiple attempts, I have been unable to reproduce this, but I watched a player on my server do these steps:
1. Carry a jersey cow using Tinkers' piggybackpack.
2. Add a hen on top with another piggybackpack.
3. Sometimes it crashes then, other times it crashes when the piggybackpacks are taken off, which releases the animals.

I understand if you can't fix the issue due to the inconsistent repro, but please at least add more logging to diagnose the issue.

### Version of Minecraft, version of Animania, Single Player or Server
Modpack: DD&SS 5.3
Forge: forge-1.12.2-14.23.5.2836-universal.jar
SpongeForge: spongeforge-1.12.2-2825-7.1.6.jar
Anamania: animania-1.12.2-1.6.2.jar
Tinker's construct: TConstruct-1.12.2-2.12.0.135.jar

### Crash report
https://gist.github.com/bluelightning32/6c3e3adef5b7b45addffcf351370c89e

### Debug log
https://gist.github.com/bluelightning32/eafa6d4e9aed880e6318d842492b5ee1

**Comment by MizuNomy** [comment]
It could just be incompatible issue or piggybag problem from Tinker's construct. Suggesting resolving problem by contacting Tinker's Construct devs.

**Comment by capnkirok** [comment]
Of course, I would recommend using Carry On for this. It was built for exactly this purpose.

**Comment by capnkirok** [comment]
And also would recommend you reach out to TC modders.
